src/main/java/io/github/thebusybiscuit/exoticgarden/PlantsListener.java

```diff
@@ -260,7 +257,7 @@ public void onDecay(LeavesDecayEvent e) {
     @EventHandler
     public void onInteract(PlayerInteractEvent e) {
         if (e.getAction() != Action.RIGHT_CLICK_BLOCK || e.getPlayer().isSneaking()) return;
-        if (SlimefunPlugin.getProtectionManager().hasPermission(e.getPlayer(), e.getClickedBlock().getLocation(), ProtectableAction.BREAK_BLOCK)) {
+        if (SlimefunPlugin.getProtectionManager().hasPermission(e.getPlayer(), Objects.requireNonNull(e.getClickedBlock()).getLocation(), ProtectableAction.BREAK_BLOCK)) {
```

Comment:
Don't do a requireNonNull here. Makes the line stupidly long and achieves no real goal. Better to just null check and return (However, it wont ever be null anyway since it's a RIGHT_CLICK_BLOCK).